examples/models/data_tables_server.py

```diff
@@ -23,10 +27,14 @@ def __init__(self):
         self.class_filter = None
 
         self.source = ColumnDataSource()
+        # A bit hacky, but now we have an index column that will carry through our slicing
+        self.df = DataFrame.from_dict(ColumnDataSource.from_df(mpg))
+        self.df.set_index('index', inplace=True)
         self.update_data()
-
-        self.document.add_root((self.create()))
-        self.session = push_session(self.document)
+        self.source.on_change('data', self.update_from_table_change)
+        # XXX: I haven't had much luck running the server in this style
+        # self.document.add_root((self.create()))
+        # self.session = push_session(self.document)
```

Comment:
I think there might be some confusion, possibly.

For apps, they *alwasy* have to be run as `bokeh serve app.py`, and should never use any of the `bokeh.client` API (e.g. `push_session`). Running `python app.py` will definitely *not* work. 

For using the client API, there are more steps. You have to run `bokeh serve` (no arguments) all by itself first in one terminal, and then separately, afterwards, run `python client_app.py`. Are you saying this process did not work? If so, that should probably result in an issue, but I'd still like to revert this to the way it was previously.